sli/slistartup.cc

```diff
@@ -173,6 +173,11 @@ SLIStartup::SLIStartup( int argc, char** argv )
   , have_libneurosim_name( "have_libneurosim" )
   , have_sionlib_name( "have_sionlib" )
   , ndebug_name( "ndebug" )
+  , mpiexec_name( "mpiexec" )
```

Comment:
See comment above on `mpiexec` vs `mpirun`

Reply:
Plus this is just the key in the status dict under which the executable name is found. I don't think many users will ever look at this.

Reply:
OK.